tests/kernels/test_awq_triton.py

```diff
@@ -0,0 +1,232 @@
+    # scales.dtype = torch.int32
+    # split_k_iters = 8
+
+    input_rows = 1
+    input_cols = 256 if small_test_size else 3584
+    input_dtype = torch.float16
+    qweight_rows = input_cols
+    qweight_cols = 32 if small_test_size else 448
+    scales_rows = qweight_rows // group_size
+    scales_cols = qweight_cols * 8
+    scales_dtype = torch.float16
+    qzeros_rows = scales_rows
+    qzeros_cols = qweight_cols
+    print(f"input_rows = {input_rows} input_cols = {input_cols}"
+          f" qweight_rows = {qweight_rows} qweight_cols = {qweight_cols}"
+          f" scales_rows = {scales_rows} scales_cols = {scales_cols}")
+
+    torch.manual_seed(2)
+    input = torch.rand((input_rows, input_cols),
+                       dtype=input_dtype,
+                       device=device)
+    qweight = torch.randint(0,
+                            torch.iinfo(torch.int32).max,
+                            (qweight_rows, qweight_cols),
+                            device=device)
+    qzeros = torch.randint(0,
+                           torch.iinfo(torch.int32).max,
+                           (qzeros_rows, qzeros_cols),
+                           device=device)
+    scales = torch.rand((scales_rows, scales_cols),
+                        dtype=scales_dtype,
+                        device=device)
+
+    # NOTE: Use to see more data and accuracy during testing.
+    # import numpy as np
+    # import sys
+    # torch.set_printoptions(precision = 3,
+    #                        threshold=10000000000000000000000000000,
+    #                        sci_mode = False)
+    # np.set_printoptions(threshold=sys.maxsize)
+
+    output_torch = awq_gemm_torch(input.cpu(), qweight.cpu(), scales.cpu(),
+                                  qzeros.cpu(), split_k_iters)
+    print(f"output_torch = {output_torch}")
+
+    output_triton = awq_gemm_triton(input, qweight, scales, qzeros,
+                                    split_k_iters)
+
+    print(f"output_triton = {output_triton}")
+    print(f"output_triton.shape = {output_triton.shape}")
+    print(f"Any infs in triton result? --> "
+          f"{torch.any(torch.isinf(output_triton))}")
+
+    diff = output_torch.cpu() - output_triton.cpu()
+    error = torch.sum(torch.sqrt(diff * diff) / torch.numel(diff))
+    print(f"error = {error}")
+
+    assert error < threshold
+
+
+def main():
+    parser = argparse.ArgumentParser(
+        description="awq_triton test driver",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument("--test")
+    known_args, unknown_args = parser.parse_known_args()
+    if known_args.test is not None:
+        if known_args.test == "dequantize":
+            test_dequantize()
+        elif known_args.test == "gemm":
+            test_gemm()
+        else:
+            print(f"Unknown test {known_args.test}")
+    else:
+        print("No test provided.")
+        parser.print_help()
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
Why not just use pytest and remove this script functionality?

Reply:
removed